### PR TITLE
Budgets home and execution redesign

### PR DESCRIPTION
**Pull request by martgnz**
Connects to #738 and #744.

### What does this PR do?

This PR redesigns the homepage of the budgets module and the execution page, adding new graphics and WIP sections.

Tasks for budgets execution page:

- [x] integrate new d3 vis
- [x] fix the new d3 vis with the custom categories
- [x] integrate last updated date
- [x] hide "custom" area button when not relevant
- [x] integrate execution % and total budgeted amount widgets
- [x] hide "functional?" area button when not relevant (some sites only have economic data)
- [x] @martgnz notice when there is no data

Tasks for budgets home:

- [x] integrate new cards
- [x] hide as pending the stories block
- [x] hide as pending receipt block
- [x] integrate the new d3 vis with bars of execution
- [x] keep the old budgets index template in a separated file (i.e. `index_old.html.erb`)
- [x] in the budget line new explorer: 
  - [x] integrate children budget line when clicking on the parent budget line
- [x] Integrate last updated date

Common to both:

- [x] Run again import scripts in staging & production so the bl updated data appears?
- [x] Integrate budges guide (remove pending blocks)

### CODE REVIEW

File: app/assets/stylesheets/css-conf.scss [hunk 1]
```diff
@@ -55,3 +55,45 @@ $color_success: #2ca25f;
...
+$f6: 1rem;
+$f7: 0.875rem;
+
+.f-mega {
+  font-size: $mega;
+}
+.f-submega {
+  font-size: $submega;
+}
+.f1 {
```
Comment (furilo) [inline app/assets/stylesheets/css-conf.scss]: I'd avoid this type of classes. Any good reason on why we need it, instead of just using the $fN variable?

Comment (martgnz) [reply app/assets/stylesheets/css-conf.scss]: It was an idea for prototyping quickly, but yes, we can delete it if it's messy.

**Comment by ferblape** [comment]
@martgnz change whatever you need to change and tell me when can we merge it.

**Comment by ferblape** [comment]
@martgnz it can happen that in a municipality we have custom data from 2016 but not for 2015. In that case the json is: `{ lines: [] }`. Can we show a message: "There's no data for this year in the visualization"?

screen shot 2017-07-13 at 15 46 09

**Comment by martgnz** [comment]
Yeah, I'll put a notice when the lines array is empty.

**Comment by ferblape** [comment]
Hey @amiedes we are showing this page to a customer, so I have restored the treemap to show it in staging  and implemented a few things such as the mega bars and the cards in the top of the page. 

Just keep working in your branch (remote the treemap and continue with the budget line cards) and don't release to staging until we have implemented all the checkes.

**Comment by martgnz** [comment]
Could you review it again, @ferblape @furilo? I made some fixes and added a warning when there aren't custom lines available.

**Review by ferblape — APPROVED** [review]
👍

**Pull request merged**